Makefile

```diff
@@ -161,6 +161,14 @@ check:
 
 .PHONY: generate
```

Comment:
To prevent someone to remove the `GO111MODULE=off` part someday when we forget about this detail. 

```suggestion
# We need to explicitly pass GO111MODULE=off to k8s.io/code-generator as it is significantly slower
# when it is executed without GO111MODULE=off - ref https://github.com/kubernetes/code-generator/issues/100.
.PHONY: generate
```

Reply:
Done